_pages/interviews.md

```diff
@@ -6,13 +6,13 @@ This is a guide for interviewers conducting engineering interviews. Everyone who
 
 To sum up:
 
-* [The golden rule: your goal is find a reason not to hire (but don’t be a jerk!)](#golden-rule)
-* When you start, [tell the candidate what to expect](#kick-off)
-* [Please follow the guides…](#follow-the-guides)
-* [… but not dogmatically: feel free to dig deep.](#dig-deep.)
-* [Take good notes: behavioral is best](#take-notes)
-* After the interview, [submit a written "hire/no-hire" recommendation, with justification](#written-recommendation)
-* [If there’s a disaster, considering ending interviews early](#pull-the-plug)
+- [The golden rule: your goal is find a reason not to hire (but don’t be a jerk!)]({{site.baseurl}}/interviews#golden-rule)
+- [Kicking off the interview]({{site.basurl}}/interviews#kick-off)
```

Comment:
Why is the {{ site.baseurl }} needed? I'd expect just /interviews#whatever to work.

Reply:
`/interviews#whatever` does not work with the HTMLProofer that we use.  I couldn't figure out why. Some of them break because they are `#very long, and punctuated! headers`, but not all of them break because of that.  The href-less links are a hack that has been present, and I just made use of.

Reply:
Huh. probably not worth burning a lot of energy on why the links aren't working. I'm OK with {{ site.baseurl }} but was just curious about why it didn't work.

I DO have a fairly strong dislike for HREF-less URLs, but again, if there's not a pretty readily apparent alternative it's probably not worth pursuing.